src/Store.ts

```diff
@@ -680,9 +679,9 @@ class Store {
     const types = type ? [type] : this.models.map(({ type }) => type)
     types.forEach((type) => {
       this.data[type] = {
-        records: observable.map(),
-        cache: observable.map(),
-        meta: observable.map()
+        records: new Map(),
+        cache: new Map(),
+        meta: new Map()
```

Comment:
we're not actually observing this anywhere, so it's just a performance hit. It would be cool to observe it though.